WordPress/Classes/ViewRelated/Reader/Select Interests/ReaderInterestsCollectionViewFlowLayout.swift

```diff
@@ -0,0 +1,204 @@
+import UIKit
+
+class ReaderInterestsCollectionViewFlowLayout: UICollectionViewFlowLayout {
+    @IBInspectable public var itemSpacing: CGFloat = 6
+    @IBInspectable public var cellHeight: CGFloat = 40
+
+    private var layoutAttributes: [UICollectionViewLayoutAttributes] = []
+    private var layoutDirection: UIUserInterfaceLayoutDirection {
+        return collectionView?.effectiveUserInterfaceLayoutDirection ?? .leftToRight
+    }
+
+    // The content width minus the content insets used when calculating rows, and centering
+    private var maxContentWidth: CGFloat {
+        guard let collectionView = collectionView else {
+            return 0
+        }
+
+        let contentInsets: UIEdgeInsets = collectionView.contentInset
+
+        return collectionView.bounds.width - (contentInsets.left + contentInsets.right)
+    }
+
+    /// The calculated content size for the view
+    private var contentSize: CGSize = .zero
+
+    override open var collectionViewContentSize: CGSize {
+        return contentSize
+    }
+
+    override func prepare() {
+        guard let collectionView = collectionView else {
+            return
+        }
+
+        let contentInsets = collectionView.contentInset
+        let isRTL = layoutDirection == .rightToLeft
```

Comment:
Is it worth having this as a property?

Reply:
Done! https://github.com/wordpress-mobile/WordPress-iOS/pull/14389/commits/0f2e8d2dba36f74ce0608c96d4b39976042b8a81